### ISSUE TITLE
For Dummies

### ISSUE THREAD

**Issue by epsy77** [jackson-c/quickpack#4]
Hey, I have little experience using the command prompt. Essentially none. Apologies if I say anything stupid. So anyway, I entered:

C:\Users\(me)>QuickPack.py "C:\Program Files (x86)\Steam\steamapps\common\Portal 2\portal2\maps\customstuff.bsp"

And it said:

QuickPack v1.4 by Jackson Cannon - https://github.com/cannon/quickpack

Reading customstuff.bsp...
Finding dependencies...
Traceback (most recent call last):
  File "C:\Users\(me)\QuickPack.py", line 467, in <module>
    main()
  File "C:\Users\(me)\QuickPack.py", line 128, in main
    newitems,deletethis = check_file(file)
  File "C:\Users\(me)\QuickPack.py", line 230, in check_file
    content = shlex.split(file.read().lower())
  File "C:\Python27\lib\shlex.py", line 279, in split
    return list(lex)
  File "C:\Python27\lib\shlex.py", line 269, in next
    token = self.get_token()
  File "C:\Python27\lib\shlex.py", line 96, in get_token
    raw = self.read_token()
  File "C:\Python27\lib\shlex.py", line 172, in read_token
    raise ValueError, "No closing quotation"
ValueError: No closing quotation

And I have no idea what to do with that error. Help?

**Comment by cannon** [comment]
It looks to me like a vmt file has an unclosed quotation ie:
`$basetexture "materials/mytexture`

I just updated the script to print the filename of which VMT has this issue. Can you let me know if this issue gets fixed?

Thanks

**Comment by epsy77** [comment]
Thanks for the quick reply! Okay, this time it went:

QuickPack v1.41 by Jackson Cannon - https://github.com/cannon/quickpack

Reading customstuff.bsp...
Finding dependencies...
    ERROR in file: materials/nature/water_moving.vmt (No closing quotation)
    ERROR in file: materials/nature/water_moving_beneath.vmt (No closing quotation)

Done. Found custom content:
    1 wav files.
    2 vmt files.

Writing to C:\Program Files (x86)\Steam\steamapps\common\Portal 2\portal2\maps\customstuff.bsp...
Done!

So it's still giving the error, but it appeared to work anyway. ~~I moved the original vmt and vtf files out of their spot, and the textures still worked when I tested the new bsp.~~ (nevermind, pink checkerboard).  
I looked through the vmt files that it named, and I couldn't find any unclosed quotation.
